src/main/java/frc/robot/subsystems/Launcher.java

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.LauncherConstants;
+import frc.robot.constants.LauncherConstants.LauncherMode;
+import frc.robot.constants.LauncherConstants.LauncherState;
+
+
+
+public class Launcher extends SubsystemBase {
+
+  private final CANSparkMax mIndexer = new CANSparkMax(LauncherConstants.INDEXER_ID, MotorType.kBrushless);
+
+  private final TalonFX mMainFlywheel = new TalonFX(LauncherConstants.MAIN_FLYWHEEL_ID);
+  private final TalonFX mRollerFlywheel = new TalonFX(LauncherConstants.ROLLER_FLYWHEEL_ID);
+
```

Comment:
Blank lines

Reply:
resolved in newest commit: "removed..." https://github.com/FRC-1294/frc2024-comp-software/tree/launcher